apps/web/src/services/auth/index.ts

```diff
@@ -3,7 +3,12 @@ import { sessions, users } from '@latitude-data/core/schema'
 import { DrizzlePostgreSQLAdapter } from '@lucia-auth/adapter-drizzle'
 import { Lucia } from 'lucia'
 
-const adapter = new DrizzlePostgreSQLAdapter(database, sessions, users)
+const adapter = new DrizzlePostgreSQLAdapter(
+  // @ts-expect-error - No idea why this is happening
+  database,
```

Comment:
No idea why started failing. Also in `main`. I'll try after merge and see if I can remove it